nkorinek/abc-classroom
# Create abc-Quickstart for classroom (#110)

* Adding files for setting up the directories

* Added dir-setup as a command line arg

* Added another arguement to directory setup, and added the template yaml in the correct place.

* Added rst documentation for functionality of abc-dir-setup

* Started fixing build docs errors

* Doc with the error

* Renamed to quickstart

* Modified accepted arguement

* Fixed minor naming issue

* Fixed a few docstrings

* Took out a return

* Reformatted with black

* Added changes suggested in github.

* quickstart changes

* Added changes to the changelog

* Pushing to help merge config file with alternate file structures

* Added tests to make sure abc-quickstart works

* Reformatted with black

* Fixed tests to be for more modern versions of python

* Reverted back to old way, since this was not the issue

* Added changes to configuration setup

* Reformatted with black

* Added changes to tests

* reformatted the test

* Minor tweaks to ensure testing works

* update course dir name

* make course name positional

* make course name positional with a default value just in case

* remove nbformat
